tenable/ot/graphql/query.py

```diff
@@ -0,0 +1,423 @@
+"""
+OT GraphQL API Queries.
+For each class, please refer to the Tenable.ot documentation website for a
+detailed explanation of the fields.
+"""
+
+ASSETS_QUERY = """
+query assets(
+  $filter: AssetExpressionsParams
+  $search: String
+  $sort: [AssetSortParams!]!
+  $limit: Int
+  $startAt: String
+) {
+  assets(
+    filter: $filter
+    search: $search
+    sort: $sort
+    first: $limit
+    after: $startAt
+  ) {
+    pageInfo {
+      endCursor
+    }
+    nodes {
+      category
+      criticality
+      customField1
+      customField2
+      customField3
+      customField4
+      customField5
+      customField6
+      customField7
+      customField8
+      customField9
+      customField10
+      description
+      details
+      family
+      firmwareVersion
+      firstSeen
+      hidden
+      id
+      ips {
+        nodes
+      }
+      lastSeen
+      lastUpdate
+      location
+      macs {
+        nodes
+      }
+      model
+      name
+      os
+      osDetails {
+        name
+        architecture
+        hotFixes {
+          nodes {
+            name
+            installDate
+            description
+          }
+        }
+        version
+      }
+      purdueLevel
+      runStatus
+      runStatusTime
+      serial
+      slot
+      superType
+      type
+      vendor
+      risk {
+        unresolvedEvents
+        totalRisk
+      }
+      ips {
+        nodes
+      }
+      macs {
+        nodes
+      }
+      backplane {
+        id
+        name
+        size
+      }
+      segments {
+        nodes {
+          id
+          name
+          type
+          key
+          systemName
+          vlan
+          description
+          assetType
+          subnet
+        }
+      }
+      revisions {
+        nodes {
+          id
+          firstSeen
+          lastSeen
+          ordinal
+          isBase
+        }
+      }
+      plugins {
+        nodes {
+          id
+          name
+          source
+          family
+          severity
+          vprScore
+          comment
+          owner
+          totalAffectedAssets
+        }
+      }
+    }
+  }
+}
+"""
+PLUGINS_QUERY = """
```

Comment:
2 blank lines.  comment detailing what this constant is

Reply:
fixed